src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -454,17 +457,32 @@ def testImportSharedFile(self, otherCls):
                        otherCls=activeTestClassesByName)
 
         def testImportHttpFile(self):
-            srcUrl, srcHash = ('https://raw.githubusercontent.com/BD2KGenomics/toil'
-                               '/12c711ff91caa5d8fb37a072bad376f447faa797/Makefile',
-                               '16a38fd7ae233616083d43d81159d4e8')
-            with self.master.readFileStream(self.master.importFile(srcUrl)) as f:
-                self.assertEqual(hashlib.md5(f.read()).hexdigest(), srcHash)
+            httpd = SocketServer.TCPServer(('', 0), StubHttpServer)
+            httpdThread = threading.Thread(target=httpd.handle_request)
```

Comment:
It might be desirable to not handle just one request, but be prepared to handle multiple requests, even though right now the import code does exactly one request. In the future, we could then support HEAD requests, multi-part downloads etc. for HTTP imports. I think this wouldn't involve more code, just using `serve_forever` instead of handle_request and `shutdown` at the end. If that turns out to be complicated, leave it the way it is.